frontend/packages/dev-console/src/components/helm/HelmReleaseResources.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { MultiListPage } from '@console/internal/components/factory';
+import { K8sResourceKind, kindForReference } from '@console/internal/module/k8s';
+import { DeploymentModel, StatefulSetModel, PodModel, SecretModel } from '@console/internal/models';
+import HelmResourcesListComponent from './HelmResourcesListComponent';
+import { flattenResources, helmReleaseResourceKindFilter } from './helm-release-resources-utils';
+import { ServiceModel } from '../../../../knative-plugin/src/models';
+
+export interface HelmReleaseResourcesProps {
+  obj: K8sResourceKind;
+}
+
+const HelmReleaseResources: React.FC<HelmReleaseResourcesProps> = ({ obj: resourceDetails }) => {
+  const { namespace } = resourceDetails.metadata;
+  const helmReleaseName = _.get(resourceDetails, 'metadata.labels.name');
```

Comment:
use optional chaining here `resourceDetails?.metadata?.labels?.['name']`

Reply:
@debsmita1 I'm guessing `metadata.labels.name` is same as the `metadata.name`? or is it different?

Reply:
it's different. `metadata.name` is the name of the secret resource & `metadata.labels.name` is the helm release name